app/controllers/activities_controller.rb

```diff
@@ -145,8 +145,9 @@ def show
 
   protected
     def set_activity
-      response = Activity.find_by_id(params[:id])
-      @activity = response.results.first
+      response = Activity.find_by_request_uuid!(params[:id])
+      @activity = response
+      puts response.inspect
```

Comment:
Is this line left over from debugging? If so it might be worth removing it before merge :)

Reply:
oops, yes.